python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1352,8 +1352,13 @@ def test_delta_binary(nrows, add_nulls, dtype, tmpdir):
 
 @pytest.mark.parametrize("nrows", delta_num_rows())
 @pytest.mark.parametrize("add_nulls", [True, False])
-@pytest.mark.parametrize("str_encoding", ["DELTA_BYTE_ARRAY"])
-def test_delta_byte_array_roundtrip(nrows, add_nulls, str_encoding, tmpdir):
+@pytest.mark.parametrize("string_len", [12, 48, 96, 128])
```

Comment:
Should this instead be four columns in the dataframe? I'm trying to test the different string copy mechanisms. Same question for nulls.

Reply:
This seems better to me, easier to see what failed from the parameter values. Unless you specifically want to test multiple copy mechanisms running at the same time.
Although this may be a question for the actual Python reviewer(s) :)

Reply:
This is finer grained, for sure, but leads to 8X the invocations of this test.  Pretty soon this library will be cuParquet plus some other stuff :rofl:

Reply:
> Pretty soon this library will be cuParquet plus some other stuff 🤣

And that's only the beginning. One day RAPIDS will stand for Reading All Parquet Inputs for Data Science.